app/components/candidate_interface/restructured_work_history_review_component.rb

```diff
@@ -16,14 +16,25 @@ def show_missing_banner?
     end
 
     def no_work_experience_rows
-      [
-        {
-          key: t('application_form.work_history.explanation.review_label'),
-          value: no_work_experience_value,
-          action: t('application_form.work_history.explanation.change_action'),
-          change_path: candidate_interface_restructured_work_history_path,
-        },
-      ]
+      if application_form.full_time_education?
```

Comment:
I'm wondering if this should really just live in it's own component. What do you think?